xwiki-platform-core/xwiki-platform-release/xwiki-platform-release-ui/src/main/resources/Release/WebHome.xml

```diff
@@ -63,19 +59,36 @@
 {{/html}}
 
 = Existing Releases =
-#set($columns = ["version", "releaseManagers", "releaseDate", "releaseNotes"])
-#set($columnsProperties = {
-    "version" : { "type" : "text" , "link" : "view"},
-    "releaseManagers" : { "html" : true },
-    "releaseDate" : { "type" : "text" }
-})
-#set($options = { 
-   "className":"ReleaseCode.ReleaseClass",
-   "selectedColumn" : "releaseDate",
-   "defaultOrder" : "desc",
-   "rowCount": 15,
-   "translationPrefix":"release.livetable."
+#set($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+       {
+         'id': 'version', 'displayer': 'link'
+       },
+       {
+         'id': 'releaseManagers', 'displayer': 'html'
+       },
+       {
+         'id': 'releaseDate', 'displayer': 'html'
+       },
+       {
+         'id': 'releaseNotes', 'displayer': 'html'
+       }
```

Comment:
These should not be needed. You can rely on the default displayer for object properties (default live data configuration coming from the live table bridge).